test/arithmetic.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const expect = require('chai').expect;
+const arithmetic = require('../lib/arithmetic.js');
+
+describe('testing module arithmetic.js', function() {
+  it('should return a number', function (){
+    expect(arithmetic).to.equal.number;
```

Comment:
You're somehow getting a false positive out of this. As far as I can tell, there's no syntax on the Chai expect docs of the form ```expect(foo).to.equal.bar```. If you're trying to test to make sure your get a number back, the right way would be:

```expect(arithmetic).to.be.a('number') ```

**HOWEVER** the above line still does not work with your test still because arithmetic itself is NOT a number. It's an object with methods attached to it. 

In other words, your module called arithmetic.js is itself an object with the methods ```arithmetic.add``` and ```arithmetic.substract```. 

So if you want full points, re-read the testing instructions, and rewrite your tests accordingly. 

You're trying to test for these methods to make sure you're getting the right output, and this test doesn't do that.